.github/workflows/static-ci-inline.yml

```diff
@@ -0,0 +1,45 @@
+name: Fontbakery CI - Inline Static Fonts
```

Comment:
What's with all the copy pasta?

All the families could've been checked with a single workflow file.

Reply:
The three independent processes were needed because when they were in a single workflow file if the process is stopped, e.g. due to the first subfamily had Fails, then the other two families are not tested.